remote/remote_state_s3.go

```diff
@@ -14,11 +15,12 @@ import (
 
 // A representation of the configuration options available for S3 remote state
 type RemoteStateConfigS3 struct {
-	Encrypt string
-	Bucket  string
-	Key     string
-	Region  string
-	Profile string
+	Encrypt   string `mapstructure:"encrypt"`
+	Bucket    string `mapstructure:"bucket"`
+	Key       string `mapstructure:"key"`
+	Region    string `mapstructure:"region"`
+	Profile   string `mapstructure:"profile"`
+	LockTable string `mapstructure:"lock_table"`
```

Comment:
It's part of the S3 backend definition: https://www.terraform.io/docs/backends/types/s3.html#lock_table